Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1

```diff
@@ -310,4 +310,15 @@ function Set-xSharePointObjectPropertyIfValueExists() {
     }
 }
 
+# Due to issues with unit testing, this function has been created so we are able to mock this function.
+function Remove-WebAppPolicy() {
```

Comment:
@BrianFarnhill Haven't been able to solve this. Left it as it is, so you can check it out. If you have some solution on how to solve this, let me know.

I have the feeling that this is caused by the fact that the module is imported in the *-TargetResource methode, overwriting the Mocked version of the method.